htlcswitch/link_test.go

```diff
@@ -5623,47 +5623,47 @@ func TestHtlcSatisfyPolicy(t *testing.T) {
 	var hash [32]byte
 
 	t.Run("satisfied", func(t *testing.T) {
-		result := link.HtlcSatifiesPolicy(hash, 1500, 1000,
+		result := link.CanForwardHtlc(hash, 1500, 1000,
```

Comment:
Test name should also be changed if the method is.

Reply:
changed